kernels/ZendEngine2/fcall.c

```diff
@@ -564,12 +564,13 @@ int zephir_call_user_function(zval **object_pp, zend_class_entry *obj_ce, zephir
 	//	fcic.function_handler->op_array.line_end = 0;
 	//}
 
-	/* fcic.initialized = 0; */
-#if PHP_VERSION_ID >= 50600
-	status = ZEPHIR_ZEND_CALL_FUNCTION_WRAPPER(&fci, &fcic, info TSRMLS_CC);
```

Comment:
@sjinks I'm pretty sure that it doesn't resolve the issue for the PHP 7.0 (ZE3) because it is ZE2. Also it seems this may cause performance regression for PHP 5.6.